qiskit_nature/problems/sampling/protein_folding/qubit_op_builder.py

```diff
@@ -0,0 +1,520 @@
+        side chain (SC) beads. In the absence of side chains, this function
+        returns a value of 0.
+
+        Returns:
+            h_scsc: Hamiltonian term consisting of side chain pairwise interactions
+        """
+        penalty_1 = self._penalty_parameters.penalty_1
+        h_scsc = 0
+        main_chain_len = len(self._peptide.get_main_chain)
+        side_chain = self._peptide.get_side_chain_hot_vector()
+        for i in range(1, main_chain_len - 3):
+            for j in range(i + 5, main_chain_len + 1):
+                if (j - i) % 2 == 0:
+                    continue
+                if side_chain[i - 1] == 0 or side_chain[j - 1] == 0:
+                    continue
+                h_scsc += self._contact_map.lower_side_upper_side[i][j] ^ (
+                    self._distance_map._first_neighbor(
+                        self._peptide, i, 1, j, 1, penalty_1, self._pair_energies
+                    )
+                    + self._distance_map._second_neighbor(
+                        self._peptide, i, 1, j, 0, penalty_1, self._pair_energies
+                    )
+                    + self._distance_map._second_neighbor(
+                        self._peptide, i, 0, j, 1, penalty_1, self._pair_energies
+                    )
+                )
+        return _fix_qubits(h_scsc)
+
+    def _create_h_short(self) -> Union[PauliSumOp, PauliOp]:
+        """
+        Creates Hamiltonian constituting interactions between beads that are no more than
+        4 beads apart. If no side chains are present, this function returns 0.
+
+        Returns:
+            h_short: Contribution to energetic Hamiltonian for interactions between beads that
+            are no
+            more than 4 beads apart.
+        """
+        main_chain_len = len(self._peptide.get_main_chain)
+        side_chain = self._peptide.get_side_chain_hot_vector()
+        h_short = 0
+        for i in range(1, main_chain_len - 2):
+            # checks interactions between beads no more than 4 beads apart
+            if side_chain[i - 1] == 1 and side_chain[i + 2] == 1:
+                op1 = self._create_turn_operators(
+                    self._peptide.get_main_chain[i + 1],
+                    self._peptide.get_main_chain[i - 1].side_chain[0],
+                )
+                op2 = self._create_turn_operators(
+                    self._peptide.get_main_chain[i - 1],
+                    self._peptide.get_main_chain[i + 2].side_chain[0],
+                )
+                coeff = float(
+                    self._pair_energies[i][1][i + 3][1]
+                    + 0.1
+                    * (self._pair_energies[i][1][i + 3][0] + self._pair_energies[i][0][i + 3][1])
+                )
+                composed = op1 @ op2
+                h_short += (coeff * composed).reduce()
+        h_short = _fix_qubits(h_short)
+
+        return h_short
+
+    # TODO verify the meaning of this term
```

Comment:
Is this an open TODO before merging the PR or will this be left for the future?

Reply:
We will ask Anton about it. We should not merge before verifying this.

Reply:
@robanton would you also help with this?